apps/terminal.py

```diff
@@ -97,6 +97,8 @@ def new_server(m):
     "shell home": "~/",
     "lefty": Key("ctrl-a"),
     "ricky": Key("ctrl-e"),
+    "clear back": Key("ctrl-u"),
+    "clear line": [Key("ctrl-e"), Key("ctrl-u")],
```

Comment:
Open to different vocab, thought about `snipline` for symmetry with editing files, but not exactly the same thing